src/utils/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Message } from '../static/sharedTypes';
+import { getQuery } from './queries';
+
+type GlobalStateKey = 'query';
+
+export function setState(key: GlobalStateKey, value: any) {
```

Comment:
when calling a promise method, better return a promise/make this method `async` - otherwise you can't `await` if you need to make sure the data was written before continuing